src/Uno.WinUIRevert/Program.cs

```diff
@@ -47,18 +47,17 @@ static void Main(string[] args)
 				Directory.Delete(dispatcherQueuePath, true);
 			}
 
+			ReplaceInFile(Path.Combine(basePath, @"src", "Directory.Build.props"), "<UNO_UWP_BUILD>false</UNO_UWP_BUILD>", "<UNO_UWP_BUILD>true</UNO_UWP_BUILD>");
+
 			// Generic replacements
 			var genericReplacements = new[] {
 				("Microsoft.UI.Xaml", "Windows.UI.Xaml"),
 				("Microsoft.UI.Composition", "Windows.UI.Composition"),
 				("Microsoft.UI.Colors", "Windows.UI.Colors"),
 				("Microsoft.UI.Text.FontWeights", "Windows.UI.Text.FontWeights"),
 				("Microsoft.UI.ColorHelper", "Windows.UI.ColorHelper"),
-				("Microsoft.UI.Xaml", "Windows.UI.Xaml"),
 				("__LinkerHints.Is_Microsoft_UI_Xaml", "__LinkerHints.Is_Windows_UI_Xaml"),
 				("__LinkerHints.Is_Windows_UI_Xaml_Controls_LayoutPanel", "__LinkerHints.Is_Microsoft_UI_Xaml_Controls_LayoutPanel"),
-				("Uno.UI.Controls.Legacy\", \"ProgressRing", "Windows.UI.Xaml.Controls\", \"ProgressRing"),
```

Comment:
It would be good if you explained why you are removing `ProgressRing` and `UI.Xaml` on this change. Jerome might know the reason but others like me reviewing this PR could learn from it.

Reply:
Microsot.UI.Xaml is duplicated.

`Uno.UI.Controls.Legacy", "ProgressRing` doesnt exist in the codebase. So it is a replacement that does nothing.

Reply:
Thanks!